api/tools.ts

```
@@ -106,44 +112,29 @@ export function registerTools(server: McpServer, apiKey: string): McpServer {
-      rawCalendarId: string;
-    }) => {
+    async ({ oauthClientId, oauthClientSecret, oauthRefreshToken, platform, rawCalendarId }) => {
       try {
-        const calendar = await baasClient.createCalendar({
-          oauthClientId,
-          oauthClientSecret,
-          oauthRefreshToken,
-          platform: platform === "Google" ? "Google" : "Microsoft",
-          rawCalendarId,
-        });
+      let createCalendarParams: CreateCalendarParams = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'createCalendarParams' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)